.github/workflows/mctc-image.yaml

```diff
@@ -0,0 +1,90 @@
+    runs-on: ubuntu-20.04
+    outputs:
+      sha_short: ${{ steps.vars.outputs.sha_short }}
+      controller_image: ${{ steps.vars-image.outputs.controller_image }}
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Get the short sha
+        id: vars
+        run: echo "sha_short=$(echo ${{ github.sha }} | cut -b -7)" >> $GITHUB_OUTPUT
+
+      - name: Get the controller image
+        id: vars-image
+        run: echo "controller_image=${{ env.IMG_REGISTRY_HOST }}/${{ env.IMG_REGISTRY_ORG }}/${{ env.IMG_REGISTRY_REPO }}:${{ steps.vars.outputs.sha_short }}" >> $GITHUB_OUTPUT
+
+      - name: Add short sha tag
+        id: add-sha-tag
+        run: |
+          echo "IMG_TAGS=${{ steps.vars.outputs.sha_short }} ${{ env.IMG_TAGS }}" >> $GITHUB_ENV
+      - name: Add latest tag
+        if: ${{ github.ref_name == env.MAIN_BRANCH_NAME }}
+        id: add-latest-tag
+        run: |
+          echo "IMG_TAGS=latest ${{ env.IMG_TAGS }}" >> $GITHUB_ENV
+      - name: Build MCTC Image
+        id: build-image
+        uses: redhat-actions/buildah-build@v2
+        with:
+          image: ${{ env.IMG_REGISTRY_REPO }}
+          tags: ${{ env.IMG_TAGS }}
+          containerfiles: |
+            ./Dockerfile
+      - name: Push to quay.io
+        id: push-to-quay
+        uses: redhat-actions/push-to-registry@v2
+        with:
+          image: ${{ steps.build-image.outputs.image }}
+          tags: ${{ steps.build-image.outputs.tags }}
+          registry: ${{ env.IMG_REGISTRY_HOST }}/${{ env.IMG_REGISTRY_ORG }}
+          username: ${{ secrets.IMG_REGISTRY_USERNAME }}
+          password: ${{ secrets.IMG_REGISTRY_TOKEN }}
+      - name: Print Image URL
+        run: echo "Image pushed to ${{ steps.push-to-quay.outputs.registry-paths }}"
+
+# TODO Update below block once glbc-deployments has been refactored. See https://github.com/Kuadrant/multi-cluster-traffic-controller/issues/27
+#  update-hcg-unstable:
+#    if: "github.repository_owner == 'kcp-dev' && github.ref_name == 'main'"
+#    name: Update HCG unstable
+#    runs-on: ubuntu-20.04
+#    needs: build
+#    steps:
+#      - uses: actions/checkout@v3
+#        with:
+#          repository: redhat-cps/glbc-deployments
+#          ref: 'main'
+#          ssh-key: ${{ secrets.ACTIONS_GITHUB_TOKEN }}
+#      - name: Install kustomize
+#        run: make kustomize
+#      - name: Setup git config
+#        run: |
+#          git config user.name "GitHub Actions Bot"
+#          git config user.email "<kuadrant.dev@redhat.com>"
+#      - name: Update glbc image
+#        run: |
+#          cd kcp-stable/root:redhat-hcg:unstable/kcp-glbc && ../../../bin/kustomize edit set image quay.io/kuadrant/kcp-glbc=${{ needs.build.outputs.controller_image }}
+#      - run: |
+#          git add ./kcp-stable/root\:redhat-hcg\:unstable/kcp-glbc/kustomization.yaml
+#          git commit -m "Update glbc image to ${{ needs.build.outputs.controller_image }}"
+#          git push origin main
```

Comment:
Personally I'd remove the commented out stuff, we can always add it when we actually need it.

Reply:
Sure, can do